L46-core-tls-credential-API.md

```diff
@@ -0,0 +1,450 @@
+L46: C-core: New TLS Credentials API
+----
+* Author(s): ZhenLian
+* Approver: markdroth, yashykt, jiangtaoli2016
+* Status: Draft
+* Implemented in:
+* Last updated: September 9, 2020
+
+## Abstract
+
+This proposal aims to provide the following security features in a new TLS credential API:
```

Comment:
nit. s/security features/features? as I believe most of the proposed features are either related to flexibility (e.g., custom server identity verification) or performance (e.g., credential reloading), but not security.